app.py

```diff
@@ -44,17 +49,27 @@ def upload_file():
         filename = secure_filename(file.filename)
         file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
         file_in = glob.glob('./uploads/*.json')
-        if (convert_from == "json" and convert_to == "png"):
-            os.system('python ./converters/' + convert_from + '2' + convert_to + '.py -i ' + file_in[0])
-        else:
-            json2glm()
+
+        supported_from_to_conversions = {
```

Comment:
Makes this a CONSTANT at the top of the file for now. It shouldn't be in the function.

Reply:
Converted the function calls - json2png(file_in) and json2glm(file_in) - as the dict values into strings and call the function with exec() now. Moved the dictionary outside the function.